src/SOS/SOS.UnitTests/ConfigFiles/Unix/Debugger.Tests.Config.txt

```diff
@@ -40,65 +36,65 @@
-      <RuntimeFrameworkVersion>$(SingleFileRuntime70Version)</RuntimeFrameworkVersion>
       <PublishSingleFile>true</PublishSingleFile>
       <BuildProjectRuntime>$(TargetRid)</BuildProjectRuntime>
+      <RuntimeFrameworkVersion>$(RuntimeVersionServicing1)</RuntimeFrameworkVersion>
     </Option>
-    <Option Condition="'$(SingleFileRuntime60Version)' != ''">
+    <Option Condition="'$(RuntimeVersionServicing2)' != ''">
       <DebuggeeBuildProcess>cli</DebuggeeBuildProcess>
       <DebuggeeBuildRoot>$(RootBinDir)/Debuggees/SingleFile</DebuggeeBuildRoot>
-      <BuildProjectFramework>net6.0</BuildProjectFramework>
-      <RuntimeFrameworkVersion>$(SingleFileRuntime60Version)</RuntimeFrameworkVersion>
       <PublishSingleFile>true</PublishSingleFile>
       <BuildProjectRuntime>$(TargetRid)</BuildProjectRuntime>
+      <RuntimeFrameworkVersion>$(RuntimeVersionServicing2)</RuntimeFrameworkVersion>
+    </Option>
+    <Option Condition="'$(RuntimeVersionServicing3)' != ''">
+      <DebuggeeBuildProcess>cli</DebuggeeBuildProcess>
+      <DebuggeeBuildRoot>$(RootBinDir)/Debuggees/SingleFile</DebuggeeBuildRoot>
+      <PublishSingleFile>true</PublishSingleFile>
+      <BuildProjectRuntime>$(TargetRid)</BuildProjectRuntime>
+      <RuntimeFrameworkVersion>$(RuntimeVersionServicing3)</RuntimeFrameworkVersion>
     </Option>
     <!--
         Default (prebuilt)
       -->
     <Option Condition="'$(RuntimeVersionLatest)' != ''">
-      <BuildProjectFramework>$(BuildProjectFrameworkLatest)</BuildProjectFramework>
       <RuntimeFrameworkVersion>$(RuntimeVersionLatest)</RuntimeFrameworkVersion>
       <SetHostRuntime>$(DotNetRoot)/shared/Microsoft.NETCore.App/$(RuntimeFrameworkVersion)</SetHostRuntime>
     </Option>
-    <Option Condition="'$(RuntimeVersion70)' != ''">
-      <BuildProjectFramework>net7.0</BuildProjectFramework>
-      <RuntimeFrameworkVersion>$(RuntimeVersion70)</RuntimeFrameworkVersion>
+    <Option Condition="'$(RuntimeVersionServicing1)' != ''">
+      <RuntimeFrameworkVersion>$(RuntimeVersionServicing1)</RuntimeFrameworkVersion>
       <SetHostRuntime>$(DotNetRoot)/shared/Microsoft.NETCore.App/$(RuntimeFrameworkVersion)</SetHostRuntime>
     </Option>
-    <Option Condition="'$(RuntimeVersion60)' != ''">
-      <BuildProjectFramework>net6.0</BuildProjectFramework>
-      <RuntimeFrameworkVersion>$(RuntimeVersion60)</RuntimeFrameworkVersion>
+    <Option Condition="'$(RuntimeVersionServicing2)' != ''">
+      <RuntimeFrameworkVersion>$(RuntimeVersionServicing2)</RuntimeFrameworkVersion>
+      <SetHostRuntime>$(DotNetRoot)/shared/Microsoft.NETCore.App/$(RuntimeFrameworkVersion)</SetHostRuntime>
+    </Option>
+    <Option Condition="'$(RuntimeVersionServicing3)' != ''">
+      <RuntimeFrameworkVersion>$(RuntimeVersionServicing3)</RuntimeFrameworkVersion>
     </Option>
     <!--
         SOS.TestExtensions
       -->
-    <Option Condition="'$(RuntimeVersion60)' != ''">
+    <Option Condition="'$(RuntimeVersionLatest)' != ''">
```

Comment:
Why is the option for SOS.TestExtensions being bumped to latest now for net9.0 and was stuck at net6.0 for net7.0 and net8.0?

Reply:
In the new scheme RuntimeVersionLatest is the only one runtime version that we can assume has been set.  There are scenarios like for servicing release testing (this is enabled by setting some CI build properties) that it is the only one set.

Reply:
Ah, so to ensure we are always testing `SOS.TestExtensions` we bump it to one we know should always be set because now `net6.0` may at some point not be included in neither `Latest` `Servicing1` `Servicing2` nor `Servicing3`?

Reply:
Yes.